composio/sdk/sdk.py

```diff
@@ -243,6 +267,8 @@ def __init__(self, api_key: str = None, base_url=get_base_url()):
         self.http_client.headers.update(
             {"Content-Type": "application/json", "x-api-key": self.api_key}
         )
+        self.local_tool_handler = LocalToolHandler()
+        self.http_handler = HttpHandler(base_url=base_url, api_key=self.api_key)
```

Comment:
We already have a http client defined on line 267, why do we need a different one?

Reply:
In HttpClient, we are making the raw calls, and thus calling /v1/action/<something> , same endpoint with arguments every time. And that is called The issue is, if some endpoint is changed, (or migrated to v2) from our hermes side, than change has effects in 3-4 places in current HttpClient style. I want to add a HttpHandler class, as a abstraction layer over HttpClient, that handles interaction to specific endpoints. 

thought?? @angrybayblade

Reply:
SDK code won't have direct contact with Hermes endpoints.

Reply:
I've taken care of those abstractions on my refactors, take a look at the client module on my branch - https://github.com/SamparkAI/composio/pull/53/files#diff-17f4a7383ae2819192cd4556b82eeb2389ca39bd3f6919573c0d8c2edd61e38fR543